public/stats-faq.md

```diff
@@ -0,0 +1,15 @@
+# STATS
+
+- __How are commits counted?__
+
+  - Commits are counted only after your PR is merged.
+  - Commits are counted if they are merged in the branch opted by mentor.Mentor is given the option to select the branch and can change it any time during the event.
+  - Commits are not linked to your github account if your email configuration is not set properly. See [this](https://github.com/kossiitkgp/kwoc-bugs/issues/22#issuecomment-747068412) for details.
```

Comment:
```suggestion
  - Commits that are not linked to your GitHub account will not be counted towards KWoC, this may happen if your email configuration is not set properly. See [this](https://github.com/kossiitkgp/kwoc-bugs/issues/22#issuecomment-747068412) for details.
```